homeassistant/components/tuya/config_flow.py

```diff
@@ -25,7 +25,9 @@
 
 RESULT_SINGLE_INSTANCE = "single_instance_allowed"
 RESULT_AUTH_FAILED = "invalid_auth"
-TUYA_ENDPOINT_BASE = "https://openapi.tuyacn.com"
+TUYA_ENDPOINT_BASE_CHINA = "https://openapi.tuyacn.com"
+TUYA_ENDPOINT_BASE_US = "https://openapi.tuyaus.com"
```

Comment:
Longterm the library should probably hold a map of country codes and endpoints.

Reply:
Both these (and more regions) are defined in the const.py file in the `TUYA_ENDPOINT` dict, why not reuse them?